pkg/api/domains/domains_test.go

```diff
@@ -0,0 +1,273 @@
+// Copyright (c) 2020, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+package domains
+
+import (
+	"encoding/json"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+	v8weblogic "github.com/verrazzano/verrazzano-crd-generator/pkg/apis/weblogic/v8"
+	"github.com/verrazzano/verrazzano-operator/pkg/controller"
+	"github.com/verrazzano/verrazzano-operator/pkg/testutil"
+	"github.com/verrazzano/verrazzano-operator/pkg/testutilcontroller"
+	"github.com/verrazzano/verrazzano-operator/pkg/types"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+// TestInit ensures that a set of domains can be obtained from the provided set of listers.
+func TestInit(t *testing.T) {
+	// GIVEN an array of clusters and a model binding pair
+	Domains = nil
+	clusters := testutil.GetTestClusters()
+	var clients kubernetes.Interface = fake.NewSimpleClientset()
```

Comment:
if you init a variable in a function, typically you use  :=    e..g
clients := fake.NewSimpleClientset()

Reply:
Strangely, though I know this is acceptable, it yielded runtime errors for my particular case so I'm sticking with the current form for the time being (at least in this case)